sfm/sfm/settings/common.py

```diff
@@ -197,4 +198,10 @@
 # sfm data space threshold to send notification email,only ends with MB,GB,TB. eg. 500MB,10GB,1TB
 DATA_THRESHOLD = env.get('DATA_VOLUME_THRESHOLD', '10GB')
 # sfm processing space threshold to send notification email,only ends with MB,GB,TB. eg. 500MB,10GB,1TB
-PROCESSING_THRESHOLD = env.get('PROCESSING_VOLUME_THRESHOLD', '10GB')
\ No newline at end of file
+PROCESSING_THRESHOLD = env.get('PROCESSING_VOLUME_THRESHOLD', '10GB')
+
+PERFORM_SERIALIZE = env.get('SFM_PERFORM_SERIALIZE', 'True') == 'True'
+SERIALIZE_HOUR = env.get('SFM_SERIALIZE_HOUR', '3')
+SERIALIZE_MINUTE = env.get('SFM_SERIALIZE_MINUTE', '0')
+
+SFM_UI_VERSION = "1.1.0"
```

Comment:
@justinlittman I think this might be 1.2.0 in order to keep the same version with current master branch. Bump version can work correctly.